src/app/(auth)/(navigationsBarLayout)/@chatDialog/page.tsx

```
@@ -1,24 +1,29 @@
 'use client';
 import { Dialog, DialogContent } from '@/components/ui/dialog';
 import { ChatRoom, ChatRoomList } from '@/features/chat';
+import useChatDialogTrigger from '@/features/chat/hooks/useChatDialogTrigger';
 import { DialogTitle } from '@radix-ui/react-dialog';
 import { useSearchParams } from 'next/navigation';
 
 export default function ChatDialog({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)